org.gitlab.neva_blyad.yat.yml

```diff
@@ -0,0 +1,286 @@
+#    org.gitlab.neva_blyad.yat.yml
+#    Copyright (C) 2024 НЕВСКИЙ БЛЯДИНА <neva_blyad@lovecry.pt>
+#                                       <neva_blyad@lovecri.es>
+#
+#    This file is part of yat.
+#
+#    yat is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, either version 3 of the License, or
+#    (at your option) any later version.
+#
+#    yat is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with yat.  If not, see <https://www.gnu.org/licenses/>.
+
+# TODO:
+# meson-perl
+# libpeas-perl
+
+app-id: org.gitlab.neva_blyad.yat
+command: yat
+
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.vala
+writable-sdk: true
+build-options:
+  prepend-path: /usr/lib/sdk/vala/bin/
+  prepend-ld-library-path: /usr/lib/sdk/vala/lib/
+  build-args:
+    - --filesystem=host
+    - --socket=session-bus
+    #- --socket=system-bus
+    - --socket=wayland
+finish-args:
+  - --devices=all # Webcam, microphone, etc
+  #- --filesystem=xdg-documents # TODO: test
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=pulseaudio # Audiocalls
+  - --socket=session-bus
+  #- --socket=system-bus
+  #- --socket=wayland # Doesn't work =( Well, I hope to fix it later
+  - --socket=x11
+
+modules:
+  #-------------------------------------------------------------------------------
+  # Sodium
+  #-------------------------------------------------------------------------------
+  - name: libsodium
+    sources:
+      - type: git
+        url: 'https://github.com/jedisct1/libsodium'
+        tag: 1.0.18
+        commit: 4f5e89fa84ce1d178a6765b8b46f2b6f91216677
+    cleanup:
+      - /include/sodium/
+      - /include/sodium.h
+      - /lib/pkgconfig/libsodium.pc
+      - /lib/libsodium.a
```

Comment:
These should be in a top level cleanup as `/include, /lib/pkgconfig, "*.a"`. Remove it from the second module too below.